This file contains an invalid character on the next line:
donï¿½
Note: don't rename file to 'test_xyz' otherwise pytest will raise
